recipe/meta.yaml

```diff
@@ -41,7 +41,7 @@ test:
     - pytest --cov graphql_relay --no-cov-on-fail --cov-report term-missing:skip-covered --cov-fail-under 91
 
 about:
-  home: https://github.com/graphql-python/graphql-relay-py
+  home: https://github.com/theacodes/cmarkgfm
```

Comment:
This appears to be an unrelated repo, could you double check that you didn't paste incorrectly?

Reply:
Thanks for the catch! I changed it back to the original home.